src/components/[guild]/RoleCard/components/EditRole/EditRole.tsx

```diff
@@ -226,22 +240,31 @@ const EditRole = ({ roleId }: Props): JSX.Element => {
               </VStack>
             </FormProvider>
           </DrawerBody>
-
-          <DrawerFooter>
-            <Button variant="outline" mr={3} onClick={onCloseAndClear}>
-              Cancel
-            </Button>
-            <Button
-              isLoading={isLoading || isSigning || isUploadingShown}
-              colorScheme="green"
-              loadingText={loadingText}
-              onClick={handleSubmit}
-              leftIcon={<Icon as={Check} />}
-              data-test="save-role-button"
-            >
-              Save
-            </Button>
-          </DrawerFooter>
+          <AnimatePresence>
+            {isDirty && (
+              <motion.div
+                initial={{ y: 300, opacity: 0 }}
+                animate={{ y: 0, opacity: 1 }}
+                exit={{ y: -300, opacity: 0 }}
+              >
+                <DrawerFooter>
```

Comment:
I think we should make a `MotionDrawerFooter` before line 57, like this:
```ts
const MotionDrawerFooter = motion(DrawerFooteR)
```
That way we wouldn't need to wrap the whole footer in a div.

Or, you could also use `LazyMotion` (see `CardMotionWrapper` for more details) ;)